### DISCUSSION TITLE
How to get the currently selected version in "[page].js"?

### DISCUSSION THREAD

**Discussion by misitebao** [facebook/docusaurus#7721] [Q&A]
I would like to know how to get the currently selected version in [page].js

**Comment by slorber** [comment]
We have an undocumented api for this, that we now consider public (ie no breaking changes on minor versions expected)

https://github.com/facebook/docusaurus/pull/7729

```js
import {useDocsPreferredVersion} from '@docusaurus/theme-common';
```
